fixture/index.js

```diff
@@ -1 +1,2 @@
-module.exports.default = '🦄';
+module.exports.default = () => '🦄';
```

Comment:
I made this a function because using `Object.assign()` on a string doesn't copy over my `rainbow` property :). That's still the only trade-off we have with using this approach, if the default export is a primitive, it will not work either. Maybe something we should document?

Reply:
Good point. Can you document that?